modules/core/shared/src/main/scala/lucuma/core/model/Target.scala

```diff
@@ -63,7 +68,10 @@ object Target extends WithGid('t'.refined) with TargetOptics {
     name:          NonEmptyString,
     ephemerisKey:  EphemerisKey,
     sourceProfile: SourceProfile
-  ) extends Target
+  ) extends Target {
+    def at(i: Instant) = none
+    def atEpoch = EphemerisCoordinates(Coordinates.Zero, Offset.Zero)
```

Comment:
Right. I considered the type class approach. It would be more flexible and avoid having an `F` on model classes.
I could experiment with that approach a bit